particle-iot/particle-cli
# Clean up keys commands

## Description

Remove usage of dfu device specs and reduce the complexity of the keys commands by supporting only one protocol per device. Also remove the need to specify the device id for `particle keys` commands if only one device is connected.

## How to Test

1. Run any `particle keys xxx` commands and `particle doctor` command on your device(s) and verify that they are working as expected. 
1. For example, To run `particle keys xxx`, run `npm start -- keys xxx` from project's root directory

## Related Issues / Discussions

https://app.shortcut.com/particle/story/122385/remove-usage-of-dfu-util-in-particle-cli

## Completeness
